tests/test_siaf_interface.py

```diff
@@ -0,0 +1,27 @@
+"""Test the functions provided by siaf_interface.
+
+Authors
+-------
+    - Johannes Sahlmann
+
+Use
```

Comment:
Is this useful? Might be confusing as it is not really what `.travis.yml` is doing (it is running all tests, right?). People who wish to run the tests can get the commands from `.travis.yml`.

Reply:
I'm a little confused by this, primarily because of my inexperience with Travis and `setup.py`. But at the moment, I think the only tests that are being run are those in `test_apt.py`, as mentioned in `setup.py`. But anyway, don't we need to have a `test_siaf_interface.py` function in order to have Travis run it?